ui.apps/src/main/content/jcr_root/apps/aemtraining/components/content/newscard/newscard.html

```diff
@@ -10,13 +10,13 @@
             <img src="${newscard.image}" alt="">
             <div class="like_dislike" style="width:400px">
                 <form action="${request.requestURI}" class="form_dislike" style="float: left; width: 130px">
-                    <button class="dislike" type="submit" name="dislike" id="dislikeId" value="false">
-                        ${newscard.disLikes}
+                    <button class="dislike" type="submit" name="dislike" value="false">
```

Comment:
Please take into account that if we use the web form here, then, upon clicking the button, the whole page is reloaded and the parameter like `like=true` is added to the address bar. 

It will stay in the address bar, and if a user just refreshes the page by e.g. pressing F5 in the keyboard, or else closes the browser and then opens it again (which leads to the automatic refreshing of the page), another like is added. Which is wrong.

Therefore, using a web form here is not a good solution. Better use an AJAX request. An AJAX request dows not make the whole page reload, and it does not change the value in the address bar.